imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -147,17 +146,18 @@ class CaseExplorer extends Component {
     }
   )
   render () {
-    const { isLoading, caseList, allNotifications, unreadNotifications } = this.props
+    const { isLoading, caseList, allNotifications, unreadNotifications, searchResult } = this.props
     const { selectedRoleFilter, sortBy, open } = this.state
     if (isLoading) return <Preloader />
-
-    const caseGrouping = this.makeCaseGrouping({
-      caseList,
-      selectedRoleFilter,
-      sortBy,
-      allNotifs: allNotifications,
-      unreadNotifs: unreadNotifications
-    })
+    const cases = this.props.searchText === '' ? caseList : searchResult
```

Comment:
searchResult correctly updates from redux

Reply:
1)searchResult exists when Iogging under render(). 
<img width="638" alt="screen shot 2018-11-26 at 9 05 13 pm" src="https://user-images.githubusercontent.com/25441733/49015862-4bf20580-f1bf-11e8-840a-8017e3877355.png">
2) search result is undefined when put as variable in makeCaseGrouping under render()
<img width="634" alt="screen shot 2018-11-26 at 9 19 09 pm" src="https://user-images.githubusercontent.com/25441733/49016450-19490c80-f1c1-11e8-8009-aef8d53527ce.png">

Reply:
I don't see any reference to `searchResult` in the function enclosed in "memoizeOne" under "makeCaseGrouping". How is it undefined there then?